cves/CVE-2016-2161.yml

```diff
@@ -66,15 +62,14 @@ fixes_vcc_instructions: |
   CVE-2011-3092.yml). Fixes and VCCs follow the same format.
 fixes:
    - commit: db5fcaac2a76d88329dbb014de0664b5b0a30bfb
-     note:
+     note: 
    - commit:
      note:
 vccs:
-  - commit:
+  - commit: c2d772c341d9b9922ade31aad299039185900b05
+    note: These memory allocation statements had been updated many times previously. The author was attempting to fix other problems when introducing this vulnerability.
```

Comment:
Could you make this text wrap?

Reply:
When this filed is parsed it won't really matter - i'm gonna leave it alone to save me a potential yaml headache.